src/plugins/intel_cpu/src/ngraph_transformations/snippets_disable_subgraph_transforms.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets_disable_subgraph_transforms.hpp"
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+SnippetsDisableSubgraphTransforms::SnippetsDisableSubgraphTransforms(Subgraphs & subgraphs) {
+    ngraph::matcher_pass_callback callback = [&](ngraph::pattern::Matcher& m) {
+        auto subgraph = std::dynamic_pointer_cast<ngraph::snippets::op::Subgraph>(m.get_match_root());
+        if (!subgraph) {
+            return false;
+        }
+        subgraph->allow_transformations(false);
+        subgraphs.emplace_back(subgraph);
+        return false;
+    };
+
+    auto subgraph = ngraph::pattern::wrap_type<ngraph::snippets::op::Subgraph>();
```

Comment:
Detail: `patterns`, `matcher_pass_callback`  are part of `ov` namespace,  I thing the legacy `ngraph` can be removed.